velox/functions/lib/string/tests/StringImplTest.cpp

```diff
@@ -282,6 +282,19 @@ TEST_F(StringImplTest, stringToCodePoints) {
       "Invalid UTF-8 encoding in characters");
 }
 
+TEST_F(StringImplTest, findNthInstanceByteIndexFromStart) {
```

Comment:
can we make sure the the same fix is also applied to strrpos
select strrpos('xxx', 'xx',2) = 1
check
findNthInstanceByteIndexFromEnd()

Reply:
findNthInstanceByteIndexFromEnd() can return the right result. Also added the test for strrpos.